<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HoughLineForm</class>
 <widget class="QWidget" name="HoughLineForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hough Line Method</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="14" column="1">
    <widget class="QSlider" name="horizontalSliderParam2L">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>250</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="buttonHoughLine">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>3. Detect lines</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="buttonCannyL">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>2. Show Canny filter</string>
     </property>
    </widget>
   </item>
   <item row="16" column="1">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <widget class="QLabel" name="label_4L">
     <property name="text">
      <string>Canny second parameter</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="buttonOpenImageL">
     <property name="text">
      <string>                    1. Open image                    </string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QLabel" name="label_3L">
     <property name="text">
      <string>Canny first parameter</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_2L">
     <property name="text">
      <string>minLineLength</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_5L">
     <property name="text">
      <string>Click to zoom in or out :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QSlider" name="verticalSliderMinLineLength">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>250</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QLabel" name="labelL">
     <property name="text">
      <string>maxLineGap</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QSlider" name="verticalSliderMaxLineGap">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>250</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>250</number>
     </property>
     <property name="tracking">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QSlider" name="horizontalSliderParam1L">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>250</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="17" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="buttonResetL">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Reset values</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSaveCannyL">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Save Canny image as ...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSaveHoughL">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Save Hough image as ...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="1" rowspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>246</width>
          <height>212</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="labelOriginalImageL">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollArea_2">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>246</width>
          <height>212</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="labelCannyL">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollArea_3">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>246</width>
          <height>212</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="labelHoughLine">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonZoomOriginalL">
       <property name="text">
        <string>Image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonZoomCannyL">
       <property name="text">
        <string>Canny</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonZoomHoughL">
       <property name="text">
        <string>Hough</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
